SPEC.md

```diff
@@ -2,33 +2,36 @@
 
 _For version information, see [VERSION](VERSION)_
 
-The "App Container" defines an image format, image discovery mechanism and execution environment that can exist in several independent implementations. The core goals include:
+"App Container" (appc) is a specification describing how applications can be packaged, distributed, and executed in a portable and self-contained way.
+The specification defines an image format, image discovery mechanism, and execution environment.
+The core goals of the specification include:
 
-* Design for fast downloads and starts of the containers
-* Ensure images are cryptographically verifiable and highly-cacheable
-* Design for composability and independent implementations
-* Use common technologies for crypto, archive, compression and transport
-* Use the DNS namespace to name and discover container images
+* Designing for fast downloads and starts of App Containers
+* Ensuring images are cryptographically verifiable and highly cacheable
+* Designing for composability and independent implementations
+* Using common technologies for cryptography, archiving, compression and transport
+* Using the DNS namespace to name and discover images
 
-To achieve these goals this specification is split out into a number of smaller sections.
+The specification consists of several key sections:
 
 1. The **[App Container Image](#app-container-image)** defines: how files are assembled together into a single image, verified on download and placed onto disk to be run.
 
-2. The **[App Container Executor](#app-container-executor)** defines: how an app container image on disk is run and the environment it is run inside including cgroups, namespaces and networking.
+2. The **[App Container Executor](#app-container-executor)** defines: how applications are grouped into execution units and the environment they are run inside (including, for example, filesystem layout, resource constraints, and networking).
 
-    * The [Metadata Server](#app-container-metadata-service) defines how a container can introspect and get a cryptographically verifiable identity from the execution environment.
+    * The [Pod Manifest](#pod-manifest) defines how one or more App Container Images are grouped into a deployable, executable unit.
```

Comment:
Hmm, "App Container Pod" to be consistent? Seems like a bit of a mouthful..
Am 18.03.2015 17:14 schrieb jonathan.boulle@coreos.com:

> Yes! I agree. I have a branch somewhere that started to do this but I
> think I got distracted 😕. Will look when in front of computer.
> Am 18.03.2015 17:12 schrieb "Charles D. Aylward" <notifications@github.com
> >:
>
>> In SPEC.md <https://github.com/appc/spec/pull/207#discussion_r26720446>:
>>
>> >
>> > -    * The [Metadata Server](#app-container-metadata-service) defines how a container can introspect and get a cryptographically verifiable identity from the execution environment.
>> > +    * The [Pod Manifest](#pod-manifest) defines how one or more App Container Images are grouped into a deployable, executable unit.
>>
>> A pod template is a producible, transmittable unit. One might be a
>> producer of images and pod templates and that's independent of a particular
>> executor. Therefore, I think the Pod Manifest could be moved out as a
>> top-level concept here.
>>
>>    1. App Container Image
>>    2. App Container Image Discovery
>>    3. Pod
>>    4. App Container Executor
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/appc/spec/pull/207/files#r26720446>.
>>
>